scripts/docker_push.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+GIT_COMMIT_ID_SHORT=`git rev-parse --short HEAD`
+if [[ -n `git status --porcelain --untracked-files=no` ]]; then
+    # uncommitted changes - dirty
+    GIT_COMMIT_ID_SHORT="$GIT_COMMIT_ID_SHORT-dirty"
+fi
+
+IMAGE_NAME="quay.io/codeready-toolchain/registration-service:$GIT_COMMIT_ID_SHORT"
+LATEST="quay.io/codeready-toolchain/registration-service:latest"
+
+echo "$DOCKER_PASSWORD" | docker login quay.io -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
just curious, how are these `DOCKER_USERNAME` and `DOCKER_PASSWORD` env vars set in the job?

Reply:
It's set manually in travis configuration for this repo - https://travis-ci.org/codeready-toolchain/registration-service/settings

Reply:
ok, cool. I knew about the encoded env vars in the `travis.yaml` file itself (see https://docs.travis-ci.com/user/encryption-keys/#usage), but I forgot that it's also doable in the console/UI